packages/components/src/components/combobox/combobox.component.ts

```diff
@@ -712,44 +696,63 @@ export default class SynCombobox extends SynergyElement implements SynergyFormCo
     this.displayInput.blur();
   }
 
+  /* eslint-disable no-param-reassign, @typescript-eslint/no-floating-promises */
   private createComboboxOptionsFromQuery(queryString: string) {
-    const optgroups: SynOptGroup[] = [];
+    this.numberFilteredOptions = 0;
+    this.isOptionRendererTriggered = true;
 
-    this.filteredOptions = this.getSlottedOptions()
-      .filter(option => this.filter(option, queryString) || queryString === '')
-      .map(option => {
-        const clonedOption = option.cloneNode(true) as SynOption;
+    // This is needed for angular. For some reason the handleDefaultSlotChange is not triggered
```

Comment:
Still not sure about this one. It does not seem to have a noticable effect when using an empty `<syn-combobox>`, but I want to point out that, whenever `createComboboxOptionsFromQuery` it will do another run of `cacheSlottedOptionsAndOptgroups` when there are no children detected. Could we detect it otherwise? Not sure.